src/service/processing_selections.py

```diff
@@ -244,7 +248,8 @@ async def save_set_to_workspace(
     workspace_id: int,
     object_name: str,
     ws_type: str,
-    description: str = None
+    service_ver: str = None,
+    description: str = None,
```

Comment:
Need to update doc string as well.

Reply:
I've got it done locally; it'll be in the next PR. Cool?